go/interpreter/lfvm/instructions_test.go

```diff
@@ -932,6 +934,80 @@ func TestSelfDestruct_ExistingAccountToNewBeneficiary(t *testing.T) {
 	}
 }
 
+func TestSelfDestruct_AccountExistsButIsNotEmpty(t *testing.T) {
```

Comment:
We have been implementing a new style lately, and I would like to adapt the name of this test to it. 
Basically, test check properties, and properties are described with a sentence that contains a verb. 
Some tests define the properties in the name of the function, and some do the same in the cases of the table. 
The sentence here does have a verb, but it does not describe an action done by opSelfDestruct, instead describes the input: 
```suggestion
func TestSelfDestruct_ReturnsSelfDestructedAndCosumesGas(t *testing.T) {
```

Reply:
And if using a table test, `ExpectedBehavior` can be the name in the map?

Reply:
Test target is, the file?  the package?  the function? 
we tend to write something like: `TestMemory_ExpandMemory_DoesExpand`

Reply:
@LuisPH3 I will try to adopt this style in the future, nice that there is a common naming scheme now.

@HerbertJordan I think it is quick to see whether the invocation is supposed to be successful as we check for `err != nil`.

Reply:
I see that this discussion has gone beyond the scope of htis PR, for the time being feel free to carry on, but let's try not to lose this discussion

Reply:
I took this discussion here: https://github.com/Fantom-foundation/Tosca/issues/883